client/jetpack-cloud/components/guided-tour/index.tsx

```diff
@@ -98,7 +105,10 @@ const GuidedTour = ( { className, tours, preferenceName }: Props ) => {
 				tour: preferenceName,
 			} )
 		);
-	}, [ dispatch, preferenceName, preference ] );
+		if ( redirectOnButtonClick ) {
+			window.location.href = redirectOnButtonClick?.toString() || '';
```

Comment:
Curious why we need to do `.toString()` here.

Reply:
We don't need it: 9bbed5d